### ISSUE TITLE
DST13 not writting LineFit parameters or Solar RA (Trac #2348)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2340]
There is a bug in the DSTExtractor for DST13 that skips setting the LineFit theta,phi parameters.
Also missing from both DST13 and DST16 is the SolarRA (right ascension in the solar reference frame).

Migrated from https://code.icecube.wisc.edu/ticket/2348
```json
{
    "status": "closed", 
    "changetime": "2019-09-08T16:02:44", 
    "description": "There is a bug in the DSTExtractor for DST13 that skips setting the LineFit theta,phi parameters.\nAlso missing from both DST13 and DST16 is the SolarRA (right ascension in the solar reference frame).", 
    "reporter": "juancarlos", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1567958564124031", 
    "component": "combo reconstruction", 
    "summary": "DST13 not writting LineFit parameters or Solar RA", 
    "priority": "major", 
    "keywords": "", 
    "time": "2019-09-05T16:30:55", 
    "milestone": "Autumnal Equinox 2019", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190905T16:31:06: juancarlos changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20190908T16:02:44: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos commented:

> In changeset 175454:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="175454"
> Closes #2340 and fixes calculation of extended sidereal time.
> ```


* juancarlos changed status from "accepted" to "closed"
